src/app/error.tsx

```diff
@@ -0,0 +1,29 @@
+'use client'
+
+import PageErrorDisplay from '@/components/PageErrorDisplay'
+
+const ErrorPage = ({
+  error,
+  reset,
+}: {
+  error: Error & { digest?: string }
```

Comment:
make a Props interface for this component

Reply:
applied in commit https://github.com/Sergei29/currency-converter-for-artfinder/pull/10/commits/a9cb3d2eacffc08e1142440a91a7c5aee12f3e31